### ISSUE TITLE
Fedora build instructions missing needed devel RPM and a required path adjustment

### ISSUE THREAD

**Issue by taw00** [BeamMW/beam#652]
I build using a bare minimum Fedora build for 28, 29, and 30 (ie. I use mock, which is the same tooling that COPR uses). -- _reference:<https://github.com/taw00/beam-rpm>_

**Problem:** Current beam docs no longer building beam on Fedora as of 2.1.4910

These are my RPM build requirements from my SPEC file...  
_This is used as an example..._
```
BuildRequires: libtool make autoconf automak
BuildRequires: gcc-c++ >= 7.0 libstdc++-static
BuildRequires: cmake >= 3.11.0
BuildRequires: gettext
BuildRequires: openssl-devel boost-devel zlib-devel
# And for the desktop stuff...
BuildRequires: qt5-qtbase-devel qt5-linguist qt5-qtsvg-devel qt5-qtwayland-devel
```

_What does this mean?_  
For a general purpose Fedora instructions you can dumb it down to (more or less divided by type)...
```
# General build tools
sudo dnf libtool make autoconf automake gettext
# C/C++ stuff
sudo dnf gcc-c++ libstdc++-static cmake boost-devel
# crypto and zlib stuff
sudo dnf openssl-devel zlib-devel
# GUI build tooling (QT in this case)
sudo dnf qt5-qtbase-devel qt5-linguist qt5-qtsvg-devel qt5-qtwayland-devel
```
And that is the minimum. The big change that broke things for me from the current docs (as of this writing) is the missing `zlib-devel`

Additionally, you guys did something odd with the pathing and I had to add a path append to get to qt5/bin in the lib directory. So...
```
export PATH=${PATH}:/usr/lib64/qt5/bin
cmake -DCMAKE_BUILD_TYPE=Release . && make -j4
```

Thanks.

**Comment by nesbox** [comment]
I updated Fedora build instructions https://github.com/BeamMW/beam/wiki/How-to-build#fedora-29
Thank you

**Comment by chapati** [comment]
Should be "sudo dnf **install**"

**Comment by taw00** [comment]
CRAP. @nesbox --- @chapati 's right. All those `sudo dnf ...` instructions should be `sudo dnf install ...` I am updated my comment.

**Comment by nesbox** [comment]
done.
